internal/store/migrations.go

```diff
@@ -600,4 +600,16 @@ var migrations = []migration{
 		 `)
 		return err
 	}},
+	{semver.MustParse("0.12.1"), semver.MustParse("0.13.0"), func(e execer) error {
+		// Add Mattermost Env column for installations.
+		_, err := e.Exec(`
+				ALTER TABLE Installation
+				ADD COLUMN MattermostEnvRaw BYTEA NULL;
```

Comment:
Do the existing `store` tests cover the case where this is `nil`? I think they should, but just to be sure.. don't want to hit that same issue I did..

Reply:
Oh, I meant to tell you that I did a bunch of testing on this and nil/null is just fine (and a little easier to read that the empty struct)